core/src/hasher/mod.rs

```diff
@@ -44,6 +48,9 @@ pub const NUM_ROUNDS: usize = Hasher::NUM_ROUNDS;
 /// Number of selector columns in the trace.
 pub const NUM_SELECTORS: usize = 3;
 
+/// Number of execution trace rows needed to fully describe one permutation of the hash function.
+pub const TRACE_CYCLE_LEN: usize = NUM_ROUNDS + 1;
```

Comment:
I find this name a bit confusing. Why not call it HASH_CYCLE_LEN as we do in other places?

Reply:
Agreed! I renamed it and updated relevant references.